### PR TITLE
fix: fix unstable test cases for flow control filter

### PR DESCRIPTION
**Pull request by ansiz**
### Issues associated with this PR

Your PR should present related issues you want to solve.

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases. And you need to show the UT result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result.

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

**Review by wangfakang — APPROVED** [review]
LGTM.

**Review by antJack — DISMISSED** [review]
LGTM

File: pkg/filter/stream/flowcontrol/stream_filter_test.go [hunk 1]
```diff
@@ -70,8 +70,15 @@ func TestStreamFilter(t *testing.T) {
 			}, nil, 0))
```
Comment (wangfakang) [inline pkg/filter/stream/flowcontrol/stream_filter_test.go:70]: need fix gethandler for the variable.
```
        variable.RegisterVariable(variable.NewBasicVariable(k, nil,
            func(ctx context.Context, variableValue *variable.IndexedValue, data interface{}) (s string, err error) {
                val := m[k]
                return val, nil
            }, nil, 0))
```

**Review by wangfakang — APPROVED** [review]

**Review by taoyuanyuan — APPROVED** [review]
LGTM

**Pull request merged**